### DISCUSSION TITLE
【求助】在开启ipv6的情况下如何优先使用ipv4的dns解析

### DISCUSSION THREAD

**Discussion by xiaobaozi233** [kkkgo/PaoPaoDNS#172] [问答和帮助]
我当前的场景非常蛋疼。
1.主路由TP LINK-XDR6088
负责拨号上网和端口转发
检测到本地有dhcp服务器后会自动关闭dhcp。

2.副路由routeros
dhcp服务器分配局域网设备ip+dns服务器+静态路由

3.PaoPaoDNS+PaoPaoGateWay
负责本地的dns解析和fakeip相关

我现在一开启ipv6后ipv6的dns服务器是tplink分配的，所有的设备就会使用ipv6的dns服务器来解析域名，paopaodns失效了。
tplink也没办法手动设置分配。
请问各位有没有什么办法可以不使用ipv6来解析地址，只使用ipv4的paopaodns。

**Comment by kkkgo** [comment]
XDR6088不是可以直接刷openwrt吗，还有硬件加速
tp自带的系统还有静态路由bug

**Comment by xiaobaozi233** [comment]
不是很想刷主路由怕刷了有问题。
现在是routeros用dhcp分配的静态路由。
主要问题是想知道同时开启ipv6和v4的情况下如何都使用paopaodns来解析

**Reply by kkkgo** [reply]
很简单，不下发IPv6的DNS，IPv6的DNS是你xdr6088下发的，这一点很显然你做不了什么。
